### PR TITLE
Fix Uncompressed Inline tar cmd

### PR DESCRIPTION
**Pull request by james-westbrook**
## Description
`Uncompressed Inline` section's `tar` command had a flag for gzip files even though that archive is uncompressed. This changes it to `tar -xv` instead so it doesn't error out.

reviewpad:summary

### CODE REVIEW

**Comment by POKT-Discourse** [comment]
This pull request has been mentioned on **Pocket Network Forum**. There might be relevant details there:

https://forum.pokt.network/t/under-review-simplify-infrastructure-deployment-and-operations/5022/1

File: doc/guides/snapshot.md
Comment (msmania) [inline doc/guides/snapshot.md]: Can you add more changes like this?

```shell
[ -d ${POCKET_DATA_DIR} ] || mkdir -p ${POCKET_DATA_DIR}
wget -c "https://pocket-snapshot.liquify.com/files/$(curl -s https://pocket-snapshot.liquify.com/files/latest.txt)" -O - \
  | tar -xv -C ${POCKET_DATA_DIR}
```

- Missing dollar sign in front of {POCKET_DATA_DIR}
- No need to download latest.txt as a file

**Review by msmania — CHANGES_REQUESTED** [review]

File: doc/guides/snapshot.md
Comment (james-westbrook) [reply doc/guides/snapshot.md]: Happy to do that, I'll push changes in a bit. 

By the way, I've been working on a script that consolidates some of the steps here and in the OPERATE A NODE docs, would love to get your take on it if you're interested: https://github.com/0xThresh/pokt-node-setup/blob/pulumi-demo/node-setup.sh

**Review by msmania — APPROVED** [review]
Thanks!

File: doc/guides/snapshot.md
Comment (msmania) [reply doc/guides/snapshot.md]: > By the way, I've been working on a script that consolidates some of the steps here and in the OPERATE A NODE docs, would love to get your take on it if you're interested: https://github.com/0xThresh/pokt-node-setup/blob/pulumi-demo/node-setup.sh

Thank you for making this effort. I like this approach with a single script.  Why don't you integrate your script to [the top page](https://github.com/pokt-network/docs-v2/blob/main/access-tutorials/README.md) of the tutorial?  You can simply add a new section like "Quick script to set up a new node" above or below the "Manual Setup Tutorial" section.

Comment (james-westbrook) [reply doc/guides/snapshot.md]: > Thank you for making this effort. I like this approach with a single script. Why don't you integrate your script to [the top page](https://github.com/pokt-network/docs-v2/blob/main/access-tutorials/README.md) of the tutorial? You can simply add a new section like "Quick script to set up a new node" above or below the "Manual Setup Tutorial" section.

Glad to hear you like it! I have a bit more testing to do before I submit that; the nginx server isn't coming up correctly yet and I need to add pocket accounts. I'll keep working on it and submit a new PR once I get further along. 

I'm also wondering how much of this stays relevant with Shannon. Do you know if there are big changes to the node setup process for Shannon?

**Comment by POKT-Discourse** [comment]
This pull request has been mentioned on **Pocket Network Forum**. There might be relevant details there:

https://forum.pokt.network/t/under-review-simplify-infrastructure-deployment-and-operations/5022/15

**Comment by Olshansk** [comment]
@0xThresh @msmania Do we still plan on merging this in?

**Comment by james-westbrook** [comment]
> @0xThresh @msmania Do we still plan on merging this in?

I would vote yes, it's still a useful update for anyone trying to create Morse nodes. I don't have write access so can't merge it myself.

**Comment by msmania** [comment]
Sorry, I forgot to click the merge button.  It's done now.

**Pull request merged**
